README.md

```diff
@@ -85,6 +90,7 @@ None
   roles:
     - ansible-role-fluentd
   vars:
+    fluentd_flags: "{% if ansible_os_family == 'FreeBSD' %}-p {{ fluentd_plugin_dir }}{% elif ansible_os_family == 'Debian' %}{% elif ansible_os_family == 'RedHat' %}{% endif %}"
```

Comment:
if -p option is mandatory for FreeBSD, __fluentd_flags should be defined in vars/FreeBSD.yml.